Adding log info in static pod, adding references in whats next section (#38229)
kubernetes/website: content/en/docs/tasks/configure-pod-container/static-pod.md

CONTAINER       IMAGE                                 CREATED           STATE      NAME    ATTEMPT    POD ID
89db4553e1eeb   docker.io/library/nginx@sha256:...    19 seconds ago    Running    web     1          34533c6729106
```
Once you identify the right container, you can get the logs for that container with `crictl`:

```shell

> # Run these commands on the node where the kubelet is running

Reviewer: You are checking the containers not the `kubelet`, right? `kubelet` runs on all nodes.

```suggestion
# Run these commands on the node where the container is running
```

CONTAINER       IMAGE                                 CREATED           STATE      NAME    ATTEMPT    POD ID
89db4553e1eeb   docker.io/library/nginx@sha256:...    19 seconds ago    Running    web     1          34533c6729106
```

> Once you identify your container You can get logs of your pod :

Reviewer 2: ```suggestion

Once you identify the right container, you can get the logs for that container with `crictl`:
```

```shell
# Run these commands on the node where the kubelet is running
crictl logs <container_id>
```
```console
10.240.0.48 - - [16/Nov/2022:12:45:49 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.47.0" "-"
10.240.0.48 - - [16/Nov/2022:12:45:50 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.47.0" "-"
10.240.0.48 - - [16/Nove/2022:12:45:51 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.47.0" "-"
```

> {{< note >}}
> To find more about how to debug using `crictl`, please visit [_Debugging Kubernetes nodes with crictl_](https://kubernetes.io/docs/tasks/debug/debug-cluster/crictl/)
> {{< /note >}}

Reviewer 2: ```suggestion
To find more about how to debug using `crictl`,  visit [Debugging Kubernetes nodes with crictl](/docs/tasks/debug/debug-cluster/crictl/).
```
(This is not important enough to need a note callout)

CONTAINER       IMAGE                                 CREATED           STATE      NAME    ATTEMPT    POD ID
f427638871c35   docker.io/library/nginx@sha256:...    19 seconds ago    Running    web     1          34533c6729106
```
## {{% heading "whatsnext" %}}

* [Generate static Pod manifests for control plane components](/docs/reference/setup-tools/kubeadm/implementation-details/#generate-static-pod-manifests-for-control-plane-components)
* [Generate static Pod manifest for local etcd](/docs/reference/setup-tools/kubeadm/implementation-details/#generate-static-pod-manifest-for-local-etcd)

> * [Debugging Kubernetes nodes with crictl](/docs/tasks/debug/debug-cluster/crictl/)

Reviewer: ```suggestion
* [Debugging Kubernetes nodes with `crictl`](/docs/tasks/debug/debug-cluster/crictl/)
```
